ts/components/settings/section/CategoryAppearance.tsx

```diff
@@ -119,6 +131,15 @@ export const SettingsCategoryAppearance = (props: { hasPassword: boolean | null
           description={window.i18n('audioMessageAutoplayDescription')}
           active={audioAutoPlay}
         />
+        <SessionToggleWithDescription
```

Comment:
I think the settings toggle for pruning Open Groups should be in the Privacy tab and not the Appearance tab but as you want.

Reply:
I like this